addons/account/migrations/9.0.1.1/pre-migration.py

```diff
@@ -212,6 +217,31 @@ def blacklist_field_recomputation(env):
     ]
 
 
+def merge_supplier_invoice_refs(env):
+    """In v8, there are 2 fields for writing references:
+    supplier_invoice_number and reference. Now in v9 there's only the last one.
+    We merge the first field content in the second one for avoiding data loss.
+    Note that previously the `reference` field has been copied for preserving
+    the original field contents.
+    """
+    openupgrade.logged_query(
+        env.cr, """
+        UPDATE account_invoice
+        SET reference = supplier_invoice_number || ' - ' || reference
```

Comment:
SET reference = supplier_invoice_number || ' - ' || %s ......
 % openupgrade.get_legacy_name('reference') ??

Reply:
No, the column exists without legacy name.

Reply:
but here https://github.com/OCA/OpenUpgrade/pull/1084/files#diff-050c291be549d31c2482669f8281e619R46 you are renaming the column...

Reply:
No, I'm copying it.

Reply:
Ok